src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,18 @@
-import React from 'react';
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <div className="PostList">
+    <ul style={{ listStyleType: 'none' }}>
+      {posts.map(post => (
+        <li key={post.id}>
+          <PostInfo post={post} />
+        </li>
+      ))}
+    </ul>
+  </div>
```

Comment:
tag `li` use in `PostInfo` component and styles add to `PostList` class
```suggestion
    <ul className="PostList">
      {posts.map(post => (
        <PostInfo post={post} key={post.id} />
      ))}
    </ul>
```